### PR TITLE
New translation on 1122

### PR DESCRIPTION
**Pull request by HENGRui6**
[oneflow.nn.CTCLoss](https://github.com/Oneflow-Inc/oneflow/blob/f4ea08520574fbc3de92d2a0a7f7cd65dd38a7d9/python/oneflow/nn/modules/loss.py#L575)
90abe5675e4416a65e3d06d19d96825

[oneflow.nn.CombinedMarginLoss](https://github.com/Oneflow-Inc/oneflow/blob/f4ea08520574fbc3de92d2a0a7f7cd65dd38a7d9/python/oneflow/nn/modules/loss.py#L884)
bf8ddae75f849e7f42d68ad598449b4

[oneflow.nn.ConstantPad1d](https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/padding.py#L152)
33a0c07d5e671f3003f02d972b7105b

[oneflow.nn.ConstantPad2d](https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/padding.py#L206)
e0f962abe73cebed709274c1a46a2d6

[oneflow.nn.ConstantPad3d](https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/padding.py#L271)
814dc5b12ca53e9b004b62bd3361bdb

[oneflow.nn.Conv1d](https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/conv.py#L74)
1c68c4d2ea23e2fc8e0b07b945d62dd
d2cbba1c09aa8a3ecbada147046a71c

[oneflow.nn.Conv2d](https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/conv.py#L231)
5af81a958c4c619bffcdb36b08832c1
e08ec1ac46979cc40a3ef4722ef0998

[oneflow.nn.Conv3d](https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/conv.py#L421)
174d914691478b52615a52f5be20ba2
7cde7d61f3d4fd179d94bc38f6b1d26

[oneflow.nn.ConvTranspose1d](https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/conv.py#L585)
877abbed85a4f81669fd0a93f719ded
718f11939a36d60f928e5d8236a0c7f

[oneflow.nn.ConvTranspose2d](https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/conv.py#L728)
8516d872802bd49c60742a5daec7e00
b82d7295ebbe74e6d96a36300317186

[oneflow.nn.ConvTranspose3d](https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/conv.py#L888)

44b0d294d7d43d3e25057cc21017edd
c388769622d8d3e153e7d63eeab56ee

[oneflow.nn.CrossEntropyLoss](https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/loss.py#L102)
a52c4cd036272bc71d938348e21ada9

### CODE REVIEW

**Comment by YakunZhou** [comment]
https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/conv.py#L121
bias(bool, optional)翻译重复了，翻译了两遍

**Comment by YakunZhou** [comment]
https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/conv.py#L770
bias中k的公式应与weight中k的公式一致

**Comment by doombeaker** [comment]
confilct 没有解决

**Comment by doombeaker** [comment]
> https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/conv.py#L770 bias中k的公式应与weight中k的公式一致

这种评论应该是针对代码行的（files changed 那个 tab 里评论，而不是当前 coversation评论），这个是昨天会议演示过的，不注意是比较不应该的。

而且，这种问题其实可以主动发现，因为可以参照其它已经合并的PR的效果（这个也是昨天会议提醒过的）

**Review by YakunZhou — COMMENTED** [review]
https://github.com/Oneflow-Inc/oneflow/blob/d47fd250fefad6ee3370a74ff91cf6dc735efb42/python/oneflow/nn/modules/conv.py#L770
bias中k的公式应与weight中k的公式一致

**Comment by doombeaker** [comment]
这个 @S882li 把冲突解决后，推动合并吧，你可以 approve 后合并的。
